envoy/config/filter/http/ext_authz/v2alpha/ext_authz.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.ext_authz.v2alpha;
+option go_package = "v2";
+
+import "envoy/api/v2/core/grpc_service.proto";
+
+import "google/protobuf/duration.proto";
+import "gogoproto/gogo.proto";
+
+// The external authorization HTTP service configuration.
+message HttpService {
+  // Sets the cluster name which the authorization request must be sent to.
+  string cluster = 1;
```

Comment:
Wondering if https://github.com/envoyproxy/data-plane-api/blob/master/envoy/api/v2/core/http_uri.proto#L9 would be appropriate here. Leaning towards saying "no", since we are probably not making arbitrary HTTP calls and can statically configure the service, but if we want to leave room for growing to a dynamic auth service resolution later, then `HttpUri` might be appropriate.

Reply:
As far as this is compatible with the previous design, which seems that it is, I think it makes sense. I'm not entirely sure about what this changes though.